### DISCUSSION TITLE
KeyError: 'location' - Initializng league command

### DISCUSSION THREAD

**Discussion by tomrulz** [cwendt94/espn-api#537] [ESPN API] [answered]
Hey all. I have been using this tool for a few years and it has been great. The new baseball season has me needing to pull my league rosters and I am having trouble using my code from last year. 

I am trying to initialize the league using the following line of code (didn't put real ID s2 or SWID in the comment): 
league = League(12345678, 2024, "espn_s2", "{SWID}")

When I run the code I get stopped at that line with the KeyError: 'location'

My guess is that i am using the wrong s2 or SWID, but using the #150 discussion I enter the values. 

Any help would be greatly appreciated.

**Comment by tomrulz** [comment]
[roster_pull_debugg.txt](https://github.com/cwendt94/espn-api/files/14898016/roster_pull_debugg.txt)

Demo code above. 

Also I notice the espn_s2 is different depending on what browser is used, could this be the problem?

**Comment by cwendt94** [comment, marked as answer]
What version of the package are you running? Make sure to upgrade to the latest version v0.35.0.

**Reply by tomrulz** [reply]
I was on v.0.30.0. THE UPGRADE to 35 worked!!! 

Sorry for the delay here I was out of town. Thanks!!!!!
